src/modules/acc/acc_mod.c

```diff
@@ -516,6 +518,12 @@ static int mod_init( void )
 		}
 	}
 
+	if( cdr_time_precision < 0 || cdr_enable > 2)
```

Comment:
Should this be
`if( cdr_time_precision < 0 || cdr_time_precision > 2)`
?

Reply:
ops :)

Reply:
fixed